Documentation/Development/Hooks/Index.rst

```diff
@@ -0,0 +1,35 @@
+.. include:: ../../Includes.txt
+
+.. highlight:: rst
+
+========
+Hooks
+========
+
+Next hooks are available in Frontend editing
+
+Frontend Editing Dropzone Modifier
+----------------------------------
+
+This is used in case you need to influence on a process of wrapping with drop zone of some specific content elements
+
+- Register your hook
+
+  .. code-block:: php
+    $GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['frontend_editing']['FrontendEditingPanel']['dropzoneModifiers'][] = \Your\NameSpace\YourClass::class;
```

Comment:
The indentation seems to be off and add <?php also, take a look at:

https://raw.githubusercontent.com/FriendsOfTYPO3/frontend_editing/master/Documentation/NewContentElements/Index.rst